sns/parity_ops.h

```diff
@@ -25,11 +25,25 @@
 #ifndef __MOTR_SNS_PARITY_OPS_H__
 #define __MOTR_SNS_PARITY_OPS_H__
 
+#ifndef __KERNEL__
+#include <isa-l/erasure_code.h>
```

Comment:
The "isa-l.h" is the top-level isa-l header file.
It is checked in Makefile.am.

It's better to use this header file.

Reply:
Done.